### PR TITLE
Release of 3.1.0

### PR DESCRIPTION
**Pull request by trevor-vaughan**
#### Pull Request (PR) description

Release of 3.0.1

### CODE REVIEW

**Comment by bastelfreak** [comment]
@trevor-vaughan can you please also generate the new changelog? The process is documented on our website: https://voxpupuli.org/docs/releasing_version/

**Comment by alexjfisher** [comment]
Most of us normally use a more direct commit message too.  eg `Release 3.0.1` vs `Prepare for 3.0.1 release`. @bastelfreak We don't actually document this anywhere though.  It's not super important, but maybe we should for consistency?

**Comment by trevor-vaughan** [comment]
@bastelfreak Well, when I tried to do that, the Changelog ended up with things that were 'pending' and didn't seem quite right.

@alexjfisher I looked at a few examples and basically followed those from other repos.

**Comment by bastelfreak** [comment]
@trevor-vaughan the generator reads the release version from metadata.json for the next release. If you first update it and afterwards generate the changelog everything should work.

**Comment by trevor-vaughan** [comment]
@bastelfreak OK, material updated

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,36 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v3.0.1](https://github.com/voxpupuli/puppet-selinux/tree/v3.0.1) (2019-12-09)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-selinux/compare/v3.0.0...v3.0.1)
+
+**Implemented enhancements:**
```
Comment (ekohl) [inline CHANGELOG.md]: Usually enhancements mean a minor version bump rather than a patch.

Comment (trevor-vaughan) [reply CHANGELOG.md]: This seems like a bug fix: https://github.com/voxpupuli/puppet-selinux/pull/312/files/74e01a0bcd32efc72ef368a9e893b22f5e0b120f#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR15

And this was a partial https://github.com/voxpupuli/puppet-selinux/pull/312/files/74e01a0bcd32efc72ef368a9e893b22f5e0b120f#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR14

I guess that the second could mean a new middle bump.

Comment (trevor-vaughan) [reply CHANGELOG.md]: Actually, thinking about this, the Release Procedures should probably be:

1. Generate CHANGELOG and figure out what version to bump
2. Bump version in metadata.json
3. Regenerate CHANGELOG

Comment (ekohl) [reply CHANGELOG.md]: That's what I always do. I also use it to review the changes. See if things are (mis)labeled as well. Don't know if it's documented.

Comment (bastelfreak) [reply CHANGELOG.md]: @trevor-vaughan if you like you can update our documentation at https://github.com/voxpupuli/voxpupuli.github.io/blob/master/_docs/releasing_version.md (that's the source for https://voxpupuli.org/docs/releasing_version/)

**Pull request merged**